checkov/sca_package_2/scanner.py

```diff
@@ -24,10 +28,35 @@ def __init__(self, pbar: ProgressBar | None = None, root_folder: str | Path | No
 
     def scan(self) -> Sequence[dict[str, Any]]:
         """run SCA package scan and poll scan results"""
-        pass
+        self.run_scan()
+        return []
 
     def run_scan(self) -> dict[str, Any]:
-        pass
+        logging.info("Start to scan package files.")
+
+        request_body = {
+            "branch": "",
+            "commit": "",
+            "path": os.path.join(bc_integration.repo_path, '') if bc_integration.repo_path else "",
+            "repoId": bc_integration.repo_id,
+            "id": bc_integration.timestamp,
+            "repositoryId": ""
+        }
+
+        response = request_wrapper(
+            "POST", f"{self._base_url}/api/v1/vulnerabilities/cli/scan",
```

Comment:
can you move `f"{self._base_url}/api/v1/vulnerabilities/cli/scan"` to the `__init__` then it can be used for the polling to0.

Reply:
sure